pandas/core/_numba/kernels/mean_.py

```diff
@@ -0,0 +1,119 @@
+"""
+Numba 1D aggregation kernels that can be shared by
+* Dataframe / Series
+* groupby
+* rolling / expanding
+
+Mirrors pandas/_libs/window/aggregation.pyx
+"""
+from __future__ import annotations
+
+import numba
+import numpy as np
+
+
+@numba.jit(nopython=True, nogil=True, parallel=False)
+def is_monotonic_increasing(bounds: np.ndarray) -> bool:
+    n = len(bounds)
+    if n == 1:
```

Comment:
I don't understand this block.  n==1 and n < 2 -> n==0?

Reply:
Good point I was able to simplify this block.